### PR TITLE
Make description editable

### PR DESCRIPTION
**Pull request by arturs105**
Related to #441 (doesn't completely close it yet)

### CODE REVIEW

File: Toggl.Foundation.MvvmCross/Helper/Color.cs [hunk 1]
```diff
@@ -52,5 +52,13 @@ public static class StartTimeEntry
 
             public static readonly MvxColor SeparatorColor = new MvxColor(181, 188, 192);
         }
+
+        public static class EditTimeEntry
+        {
+            //This should be the same as pinkishGrey, but iOS makes the color a bit darker,
+            //when applied to UITextField.AttributedPlaceholder, so this is made a bit
+            //lighter than the actual color.
+            public static readonly MvxColor Placeholder = new MvxColor(215, 215, 215);
```
Comment (willsb) [inline Toggl.Foundation.MvvmCross/Helper/Color.cs]: If you are including iOS specific remarks, why is this in the shared portion of the code?

File: Toggl.Daneel/ViewControllers/EditTimeEntryViewController.cs [hunk 2]
```diff
@@ -30,6 +35,18 @@ public override void ViewDidLoad()
 
             resizeSwitch();
 
+            var placeholderAttributes = new UIStringAttributes
```
Comment (willsb) [inline Toggl.Daneel/ViewControllers/EditTimeEntryViewController.cs]: I'd move all this view manipulation related code away from ViewDidLoad and into its own method, like we are doing in every other VC (similar to what you did on the VC for `resizeSwitch`)

File: Toggl.Foundation.MvvmCross/Helper/Color.cs (hunk 1 above)
Comment (arturs105) [reply Toggl.Foundation.MvvmCross/Helper/Color.cs]: Forgot that this is shared. Will change

File: Toggl.Daneel/ViewControllers/EditTimeEntryViewController.cs (hunk 2 above)
Comment (arturs105) [reply Toggl.Daneel/ViewControllers/EditTimeEntryViewController.cs]: 👍

**Comment by amulware** [comment]
Please reference appropriate issues in the description.

**Pull request merged**